src/pages/home/components/order-category.tsx

```
@@ -22,7 +28,10 @@ export const OrderCategory = ({ status, title }: Props) => {
   const { token = "" } = user;
 
   const fetchOrders = async (pg: number) => {
-    const res = await getOrders(token, pg, status);
+    console.log(user);
+    const res = true
```

Biome

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)